nextflow_schema.json

```diff
@@ -42,6 +53,117 @@
                 }
             }
         },
+        "ccs_options": {
+            "title": "ccs options",
+            "type": "object",
+            "description": "Set options for each program",
+            "default": "",
+            "properties": {
+                "chunk": {
```

Comment:
If possible, add a more descriptive description what the parameter does. Just one line and in the help text you can add the full info. Like you did for TAMA collapse

Reply:
I updated this description to match the option set